public/app/features/dashboard/components/PanelEditor/types.ts

```diff
@@ -15,10 +15,14 @@ export enum DisplayMode {
   Fill = 0,
   Fit = 1,
   Exact = 2,
+  Inspect = 3,
+  JSON = 4,
 }
 
 export const displayModes = [
   { value: DisplayMode.Fill, label: 'Fill', description: 'Use all avaliable space' },
   { value: DisplayMode.Fit, label: 'Fit', description: 'Fit in the space keeping ratio' },
   { value: DisplayMode.Exact, label: 'Exact', description: 'Same size as the dashboard' },
+  { value: DisplayMode.Inspect, label: 'Inspect', description: 'Show the panel inspector content' },
```

Comment:
Not sure this should be a display mode. Maybe just an inspect "button" in the toolbar instead?